browser/js/global.js

```diff
@@ -279,6 +268,51 @@ window.app = {
 		},
 	};
 
+	global.touch = {
+		/// a touchscreen event handler, supports both DOM and hammer.js events
+		isTouchEvent: function(e) {
+			if (L.Browser.cypressTest && global.L.Browser.mobile) {
+				return true; // As cypress tests on mobile tend to use "click" events instead of touches... we cheat to get them recognized as touch events
+			}
+
+			if (!e.pointerType) {
+				return !(e instanceof MouseEvent);
+			}
+
+			return e.pointerType === 'touch' || e.pointerType == 'pen' || e.pointerType == 'kinect';
+		},
+
+		/// a decorator that only runs the function if the event is a touch event
+		touchOnly: function(f) {
+			return function(e) {
+				if (!global.touch.isTouchEvent(e)) return;
```

Comment:
nit': I tend to really dislike this formatting.
```suggestion
				if (!global.touch.isTouchEvent(e)) {
					return;
				}
```

also elsewhere

Reply:
it does.

![image](https://github.com/CollaboraOnline/online/assets/114441/a583e1bf-c030-46b8-a609-add1c6e51e8e)

note: none of this is blocking either.